Update prerequisites.rst (#7764)
IQSS/dataverse: doc/sphinx-guides/source/installation/prerequisites.rst

Our sample init script and systemd service file linked above tell Solr to only listen on localhost (127.0.0.1). We strongly recommend that you also use a firewall to block access to the Solr port (8983) from outside networks, for added redundancy.

To start Solr Apache in external use mode, you must access the Solr example directory:

> # cd /usr/local/solr-7.7.2/bin/

Reviewer: ```suggestion
# cd /usr/local/solr/solr-8.8.1/bin
```

> # ./solr start -j "-Djetty.host = DNS” -force

Reviewer: ```suggestion
# ./solr start -j "-Djetty.host = DNS" -force
```

> Right after that you should use the iptables blocking rule

Reviewer: ```suggestion
Right after that you should use the iptables rules below.
```

(Only one rule blocks. The other accepts.)

> # iptables -A INPUT -s ip address / 32 -p tcp -m tcp --dport 8983 -j ACCEPT

Reviewer: This "ip address" is for the Dataverse server, right? To make this more clear, perhaps it could be IP_OF_DATAVERSE_SERVER.

Author: yes, IP_OF_DATAVERSE_SERVER i  fully agree.

> # iptables -A INPUT -p tcp -m tcp --dport 8983 -j DROP

Reviewer: For all of these commands please use the format for "usermod" and "sudo" in the screenshot below. This ensures consistency and prevents quotes from being turning into smart quotes.

<img width="976" alt="Screen Shot 2021-04-08 at 11 15 34 AM" src="https://user-images.githubusercontent.com/21006/114052367-fe403a00-985b-11eb-903c-fd3fe657bf47.png">

Author: I agree, not least because it validates that the execution will be correctly. use command sudo.

> As a result, your application will reject Solr's external access, assuring connections only through Dataverse.

Reviewer: ```suggestion
As a result, Solr's external access will be blocked except from your Dataverse installation.
```

If you want to upload Solr Apache without a GUI prompt, use it as follows:

# ./solr start -j "-Djetty.host = hostaname" -noprompt

> With that application will work on localhost without access to GUI.

Reviewer: I'm not sure this part about the GUI is necessary. If you're saying it's related to securing Solr, you might want to mention this. Finally, do you mean "upload"? Maybe "access" instead? Or "use"?

Author: When the "./solr start -j" part is mentioned -Djetty.host = hostaname "-noprompt"

it is more for security reasons in the local environment so that the user does not access the solr locally and make undue changes.
